### ISSUE TITLE
☂️ Improve Apricot accessibility

### ISSUE THREAD

**Issue by patricklafrance** [gsoft-inc/sg-orbit#527]
## Design principles

Craft user-centric solutions

## Description

[Stats show up to 10-15% of users could be affected by accessibility problems](https://www.interactiveaccessibility.com/accessibility-statistics). Being part of GSoft, a company who promotes inclusivity, we believe ShareGate softwares should be inclusive and find the right balance between accessibility/quality and development effort.

Furthermore, as accessibility adoption increase among websites, users tend to expect it. Moving to an accessible app is a long process, and we believe we should be ahead of the curve.

From a technical standpoint, adopting browser makers specifications, also ensure we get long term free support on multiple devices and platforms.

We aim to target a [subset of WCAG AA specifications](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&versions=2.0&showtechniques=211%2C247&levels=aaa) but it still have to be defined.

## Related issues

- [ ] Define ShareGate accessibility standards
- [x] Adapt ShareGate colors to match WCAG contrast specifications (https://github.com/gsoft-inc/sg-orbit/issues/525)
- [ ] Write Orbit components accessibility documentation 
- [ ] Improve Apricot developpers awareness for accessibility 
- [x] Add a11y automated tools to Orbit & Apricot (ex. eslint plugin, Storybook addon, ...)
